README.md

```diff
@@ -1,29 +1,106 @@
+ 2. Follow the setup instruction of the [CNTK Fast-RCNN tutorial](https://github.com/Microsoft/CNTK/wiki/Object-Detection-using-Fast-R-CNN#setup) (*Note: Fast-RCNN  is only supported for Linux with python 3.4 and not 3.5*)
+ 3. Download and Extract the app [release package](https://github.com/CatalystCode/CNTK-Object-Detection-Video-Tagging-Tool/releases)
+ 4. Configure CNTK-Config.json with the following properties
+ 
+  ```json
+   {
+    "cntkPath" : "{CNTK Path default is c:/local/cntk}", 
+   }
+  ```
+ 5. Run the app
+ 
+-----------
+<a name="taggingjob"></a>
+## Tagging Job 
+ 1. Load a mp4 video file by either dragging it into the app or clicking and selecting it.
+  
+    <img src="/media/2_load.jpg" alt="Home Page" height="426" width="695"/>
+   
+ 2. Configure the tagging job specifying the following preferences
+ 
+    <img src="/media/3_Job_Configuration.jpg" alt="Home Page" height="586" width="685"/>
+ 
+    **Frame Extraction Rate:** The number of frames to tag per a second of video.<br>
+    
+    **Tagging Region Type:**  The type of bounding box for tagging regions.<br>
+      - "Rectangle": Tag bounding boxes of any dimension.
+      - "Square": Tag bounding boxes of auto-fixed dimensions.
+      - "Point": Tag a point (This is useful for object recognition but not object detection)
+      
+    **Export Frames Until:** How far will the export operation proceed.<br>
+      - "Last Tagged Region": Exports frames up until the last frame that contain tags.
+      - "Last Visited Frame": Exports frames up until the last frame that the user explicitly visited.
+      - "Last Frame": Exports all video frames.
+      
+    **Output directory:** Directory path for exporting training data.<br>
+    
+    **Model Path:** File path of the trained Fast-RCNN model file (Optional).<br>
+    
+    **Labels:** The labels of the tagged regions (Ex. Cat, Dog, Horse, Person).<br>
+  
+ 3. Tag the video frame by frame.
+ 
+    <img src="/media/4_Tagging_Job.jpg" alt="Home Page" height="586" width="685"/>
+    
+    **Tagging**: To add a region to a frame simply click and drag a bounding box around the desired object. Then move or resize the region till it perfectly fits the desired area.
+     - Selected regions appear as red ![red](https://placehold.it/15/f03c15/000000?text=+) and unselected regions will appear as blue ![#1589F0](https://placehold.it/15/1589F0/000000?text=+).
+     - A tag can be assigned to a region by clicking on a region and selecting the desired tag from the label toolbar at the bottom of the tagging control. 
+     - To clear all tags on a given frame click the clear tags button to the right navigation buttons on the video tagging control 
+    
+    **Navigation**: Users can navigate video frames by using the left and right arrow keys or clicking the forward and back buttons on the control.
```

Comment:
I suggest to add images of the buttons to make it clearer. It's crazy how little graphics can make the README look much better